rc/tools/git.kak

```diff
@@ -71,22 +77,27 @@ define-command -params 1.. \
     if [ $kak_token_to_complete -eq 0 ]; then
         printf "add\nrm\nreset\nblame\ncommit\ncheckout\ndiff\nhide-blame\nhide-diff\nlog\nnext-hunk\nprev-hunk\nshow\nshow-branch\nshow-diff\ninit\nstatus\nupdate-diff\ngrep\n"
     else
+        # Try cd-ing into the git work tree if exists to make autocompletion slightly more robust
+        cd "$kak_opt_git_work_tree_top_level" >/dev/null 2>&1
```

Comment:
I expect completion to use exactly the same cwd as the command itself. It
is surprising that we use the value from a previous :git invocation here.

Reply:
That was a consequence of my stateful variable. But I see this comment was older than others so you realized that by then.